src/qt/bitcoingui.cpp

```diff
@@ -1425,25 +1425,40 @@ void BitcoinGUI::timerfire()
 
 }
 
-QString BitcoinGUI::GetEstimatedTime(unsigned int nEstimateTime)
+QString BitcoinGUI::GetEstimatedStakingFrequency(unsigned int nEstimateTime)
 {
     QString text;
-    if (nEstimateTime < 60)
+
+    if (!nEstimateTime)
     {
-            text = tr("%n second(s)", "", nEstimateTime);
+        text = tr("undefined");
```

Comment:
I think you intended to put a `return` after this branch.

Reply:
yep